src/ExpandablePicker/ExpandablePicker.js

```diff
@@ -266,6 +267,30 @@ module.exports = kind(
 	},
 
 	/**
+	*  'multiSelectCurrentAccesibilityLabel()' can be overridden by subkinds, such as moon.DayPicker.
+	*  Returns accessibilityLabel for selected items by concatenate accessibilityLabel of each items.
+	*
+	* @protected
+	*/
+	multiSelectCurrentAccesibilityLabel: function () {
+		if (!this.multipleSelection) {
+			return;
```

Comment:
We also consider single selection case. Is there any other place to take care of single selection case? I can't find